CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
   - Adds a _Jump to Reference_ as an `alt` modifier to the _Jump to HEAD_ button to jump to the selected branch or tag
 - Adds support deep link documentation &mdash; closes [#3399](https://github.com/gitkraken/vscode-gitlens/issues/3399)
 - Adds a pin status icon to Launchpad items when pinned
+- Shows GitLab merge requests in the _Launchpad_ when GitLab integration is connected
```

Comment:
@sergeibbb Oh, one thing I noticed - this line is in the 15.2.1 section. It should be in the "Unreleased" section (you may need to add an "Added" category to that section).